### ISSUE TITLE
Segfault when printing AST on module parse error

### ISSUE THREAD

**Issue by Dandigit** [c2lang/c2compiler#95]
Currently, when c2c is given this invalid C2 code:
```
module module;
```
...and asked to print the AST by providing the `-a0` flag:
```
c2c -f file.c2 -a0
```
...the error is correctly reported, but when c2c tries to print the AST, it results in a segfault:
```
module module;
       ^
---- AST (module=) /home/dan/invalid.c2 ----

Segmentation fault (core dumped)
```

I'm on Ubuntu 18.04.1 LTS.

**Comment by Dandigit** [comment]
I've provided a quick fix in #96.

**Comment by bvdberg** [comment]
indeed, merged the fix! Thx
